source/FormationPositioner.h

```diff
@@ -0,0 +1,87 @@
+/* FormationPositioner.h
+Copyright (c) 2019 by Peter van der Meer
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#ifndef FORMATION_POSITIONER_H_
+#define FORMATION_POSITIONER_H_
+
+#include "Angle.h"
+#include "Body.h"
+
+#include <map>
+
+class FormationPattern;
+class Ship;
+
+
+
+// Represents an active formation for a set of spaceships. Assigns each ship
+// to a position (Point) in the formation.
+class FormationPositioner{
+public:
+	// Initializer based on the formation pattern to follow.
+	FormationPositioner(const Body *formationLead, const FormationPattern *pattern): formationLead(formationLead), pattern(pattern), direction(formationLead->Facing()) {}
```

Comment:
The inline constructors were also used in CollissionSet.h, EsUuid.h, MapPanel.h, Panel.h and Ship.h. I assumed that this was done for readability (to avoid constructors in the cpp files that are just empty), but I don't mind to move the empty body to a cpp file (which indeed seems to be a little bit more correct).